components/StyledButton/index.js

```diff
@@ -2,13 +2,23 @@ import styled from "styled-components";
 
 export const StyledButton = styled.button`
   padding: 0.8rem;
+  margin: ${(props) =>
+    props.$variant === "delete" || props.$variant === "edit" ? "0" : "1rem 0"};
   border-radius: 0.6rem;
-  color: black;
-  text-decoration: none;
   font-weight: bold;
-  border: none;
+  color: white;
+  border: ${(props) =>
+    props.$variant === "delete" || props.$variant === "edit" ? " " : "none"};
   cursor: pointer;
-  background-color: ${(props) => {
-    return props.$variant === "delete" ? "white" : "lightsalmon";
-  }};
+  background: ${(props) =>
+    props.$variant === "delete" || props.$variant === "edit"
```

Comment:
I think what you mean is the save-button that gets the gradient background? Maybe you want to change the $variant from "edit" to "save" just to make it a bit clearer.

Reply:
the target of this is not the "Save"-buttton in the form, but the "Edit"-Button (edit-icon) that leads to the Form which enables editing and saving.

Reply:
The edit button does not have a gradient background. But the save button has.